docker-compose/docker-compose.yml

```diff
@@ -40,7 +40,7 @@ services:
   # WHISK CONTROLLER
   controller:
     image: ${DOCKER_IMAGE_PREFIX:-openwhisk}/controller
-    command: /bin/sh -c "exec /init.sh 0 >> /home/owuser/controller-local_logs.log 2>&1"
+    command: /bin/sh -c "exec /init.sh 0 >> /logs/controller-local_logs.log 2>&1"
```

Comment:
does the process have access to write in `/logs` actually ?  Alternatively we can do https://github.com/apache/incubator-openwhisk-devtools/pull/259/commits/b9753eb30239345fe6e2f64e6d3e8e2b70025618#diff-cb44929492b826b5b16656db6a5a26a4R76